pdchaosazure/vmss/actions.py

```diff
@@ -307,10 +309,18 @@ def burn_io(filter_vmss: str = None,
 
     duration : int, optional
         Duration of the stress test (in seconds) that generates high disk I/O operations. Defaults to 60 seconds.
+
+    size : int
+        Size of the file in megabytes that is written to. Defaults to 30 MB.
```

Comment:
Can we describe it a bit better.
What is this size responsible for? Why user would like to change it?